server/helpers/partyHelpers.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable eol-last */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable indent */
+
+class Party {
+  constructor() {
+    this.parties = [];
+  }
+
+  createParty(party) {
+    const newParty = {
+      id: this.parties.length + 1,
```

Comment:
Pls use the field Name format as specified in the ADC docs 
<img width="544" alt="screen shot 2019-01-29 at 12 59 59 am" src="https://user-images.githubusercontent.com/26818808/51877037-ebbc4300-236a-11e9-8b61-6499fcf77d38.png">